packages/touchpoint-ui/src/components/RiveAnimation.tsx

```diff
@@ -25,9 +25,13 @@ function resolveCssVariable(
   ];
 }
 
-export const RiveAnimation: FC<{ restored: boolean }> = ({ restored }) => {
+export const RiveAnimation: FC<{ restored: boolean; volumeLevel: number }> = ({
+  restored,
+  volumeLevel,
+}) => {
   const samplerRef = useRef<HTMLCanvasElement | null>(null);
   const ref = useRef<HTMLCanvasElement | null>(null);
+  const voiceInputRef = useRef<{ value: number }>({ value: 0 });
```

Comment:
I don't get what's happening with this ref, it's just defined, written to, and not read anywhere. I also don't see the voiceInput rive variable being written to.

Reply:
Oooooh I get it, clever 😎